### PR TITLE
Navbar enhancements

### PR DESCRIPTION
**Pull request by ahmad856**
sasti nav bar ka code

### CODE REVIEW

File: src/components/header/components/dropdown/LabServicesDropDown.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {Button} from "@material-ui/core";
+import Popper from "@material-ui/core/Popper";
+import Grow from "@material-ui/core/Grow";
+import Paper from "@material-ui/core/Paper";
+import ClickAwayListener from "@material-ui/core/ClickAwayListener";
+import ButtonGroup from "@material-ui/core/ButtonGroup";
+import {makeStyles} from "@material-ui/core/styles";
```
Comment (rehan99000) [inline src/components/header/components/dropdown/LabServicesDropDown.js]: group imports

File: src/components/header/components/dropdown/LabServicesDropDown.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                                    <Button href="/appointment">Book Appointment</Button>
+                                </ButtonGroup>
+                            </ClickAwayListener>
+                        </Paper>
+                    </Grow>
+                )}
+            </Popper>
+        </div>
+    );
+}
```
Comment (rehan99000) [inline src/components/header/components/dropdown/LabServicesDropDown.js]: add new line

File: src/components/header/components/dropdown/ProviderServicesDropDown.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {Button} from "@material-ui/core";
+import Popper from "@material-ui/core/Popper";
+import Grow from "@material-ui/core/Grow";
+import Paper from "@material-ui/core/Paper";
+import ButtonGroup from "@material-ui/core/ButtonGroup";
+import {SupplyOrderFormDialog} from "../formdialog/SupplyOrderFormDialog";
+import {makeStyles} from "@material-ui/core/styles";
+import ClickAwayListener from "@material-ui/core/ClickAwayListener";
```
Comment (rehan99000) [inline src/components/header/components/dropdown/ProviderServicesDropDown.js]: group imports

File: src/components/header/components/dropdown/ProviderServicesDropDown.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                                    <SupplyOrderFormDialog/>
+                                </ButtonGroup>
+                            </ClickAwayListener>
+                        </Paper>
+                    </Grow>
+                )}
+            </Popper>
+        </div>
+    );
+}
```
Comment (rehan99000) [inline src/components/header/components/dropdown/ProviderServicesDropDown.js]: add new line

File: src/components/header/components/formdialog/SupplyOrderFormDialog.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {Button} from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import TextField from '@material-ui/core/TextField';
+import DialogActions from '@material-ui/core/DialogActions';
```
Comment (rehan99000) [inline src/components/header/components/formdialog/SupplyOrderFormDialog.js]: group imports

**Review by rehan99000 — CHANGES_REQUESTED** [review]

File: src/components/header/components/dropdown/LabServicesDropDown.js (hunk 1 above)
Comment (ahmad856) [reply src/components/header/components/dropdown/LabServicesDropDown.js]: ker deya gandu

File: src/components/header/components/dropdown/LabServicesDropDown.js (hunk 2 above)
Comment (ahmad856) [reply src/components/header/components/dropdown/LabServicesDropDown.js]: ye bhi

File: src/components/header/components/dropdown/ProviderServicesDropDown.js (hunk 3 above)
Comment (ahmad856) [reply src/components/header/components/dropdown/ProviderServicesDropDown.js]: ye bhi

File: src/components/header/components/dropdown/ProviderServicesDropDown.js (hunk 4 above)
Comment (ahmad856) [reply src/components/header/components/dropdown/ProviderServicesDropDown.js]: ye bhi

File: src/components/header/components/formdialog/SupplyOrderFormDialog.js (hunk 5 above)
Comment (ahmad856) [reply src/components/header/components/formdialog/SupplyOrderFormDialog.js]: ye bhi

**Pull request merged**